tests/acceptance/semgrep-rules-repo/local-config-full-scan.err

```diff
@@ -10,9 +10,5 @@ Note that no baseline-ref was passed and detected environment is not Github or G
 === looking for current issues in 1473 files
 | 2 current issues found
 | No ignored issues found
-| 2 current issues found
```

Comment:
Old code was printing per chunk instead of once in the end

Reply:
@brendongo Does this address https://github.com/returntocorp/semgrep-action/issues/159?

Reply:
Oh yes it does. Didn't realize there was an open issue

Reply:
I was hoping we could keep that behavior in some form and just add lines saying "Processing batch 4 of 8" in-between, so that the agent doesn't spend minutes at a time with no output. But this seems difficult with this new architecture so might as well do it this way 👍

Reply:
Yay, Christmas CAN come early 🙌